packages/flutter/lib/src/material/text_field.dart

```diff
@@ -172,22 +172,14 @@ class _TextFieldSelectionGestureDetectorBuilder extends TextSelectionGestureDete
 /// when it is no longer needed. This will ensure we discard any resources used
 /// by the object.
 ///
-/// {@tool snippet}
-/// This example shows how to create a [TextField] that will obscure input. The
+/// ## Obscured Input
+///
+/// {@tool dartpad}
+/// This example shows how to create a [TextField] that will obscured input. The
```

Comment:
```suggestion
/// This example shows how to create a [TextField] that will obscure input. The
```

Sorry, I gave you a bad suggestion before.